aws-amp-integration.md

```diff
@@ -9,136 +9,271 @@ Kubecost leverages the open-source Prometheus project as a time series database
 * [Amazon Managed Prometheus (AMP)](https://docs.aws.amazon.com/prometheus/latest/userguide/what-is-Amazon-Managed-Service-Prometheus.html)
 * [AMP IAM permissions and policies](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-and-IAM.html)
 
-## Install AMP
+## Architecture
+The architecture of this integration is similar to Amazon EKS cost monitoring with Kubecost, which is described in the previous blog post, with some enhancements as follows:
 
-### Prerequisites
+In this integration, an additional AWS SigV4 container is added to the cost-analyzer pod, acting as a proxy to help query metrics from Amazon Managed Service for Prometheus using the AWS SigV4 signing process. It enables passwordless authentication to reduce the risk of exposing your AWS credentials.
```

Comment:
Is there any reason why you're no longer abbreviating Amazon Managed Prometheus to AMP in this doc?

Reply:
Per AWS marketing requirement, I keep it that way in all documentation I wrote for AWS integration to not fix it later.